src/GraphControl/Control/Grapher.cpp

```diff
@@ -1040,6 +1043,34 @@ void Grapher::OnGraphBackgroundPropertyChanged(Windows::UI::Color /*oldValue*/,
     }
 }
 
+
+void Grapher::OnGridLinesColorPropertyChanged(Windows::UI::Color /*oldValue*/, Windows::UI::Color newValue)
```

Comment:
This one has the namespaces because of the 2 Color types Windows::UI::Color and Graphing::Color. The other methods for background and axis color also have the namespaces.